packages/ember-metal/tests/main_test.js

```diff
@@ -32,3 +32,14 @@ QUnit.test('SEMVER_REGEX properly validates and invalidates version numbers', fu
   validateVersionString('1.11', false);
 });
 
+QUnit.test('Ember.keys is deprecated', function() {
+  expectDeprecation(function() {
+    Ember.keys({});
+  }, 'Ember.keys is deprecated in-favour of Object.keys');
+});
+
+QUnit.test('Ember.create is deprecated', function() {
+  expectDeprecation(function() {
+    Ember.keys({});
```

Comment:
The test says `Ember.create is deprecated` but you are invoking `Ember.keys({})` here (likely copy/paste error).